schema.sql

```diff
@@ -0,0 +1,26 @@
+-- DROP DATABASE IF EXISTS Player;
```

Comment:
Hey if this file is not going to be used it would be better to have it deleted

Reply:
You're totally right, I forgot about that file.  Thanks for the catch.  It has beeen removed